ComponentSnapshotTestCase/CKComponentSnapshotTestCase.h

```diff
@@ -13,10 +13,10 @@
 
 #import <FBSnapshotTestCase/FBSnapshotTestCase.h>
 
-#define CK_AT_LEAST_IOS7_1 (kCFCoreFoundationVersionNumber >= 847.24)
```

Comment:
There aren't any reference images for iOS 7 so I figured I could remove this as well.